tools/data-api/internal/repositories/services.go

```diff
@@ -337,18 +338,15 @@ func parseModelFromFilePath(filePath string) (*ModelDetails, error) {
 		return nil, fmt.Errorf("unmarshaling %q: %+v", filePath, err)
 	}
 
-	typeHintIn := ""
 	fieldDetails := make(map[string]FieldDetails)
 	for _, field := range model.Fields {
 		fieldDetail := FieldDetails{
-			IsTypeHint: field.ContainsDiscriminatedTypeValue,
-			JsonName:   field.JsonName,
-			Optional:   field.Optional,
-			Required:   field.Required,
-			// TODO remove Default attribute throughout the models since this isn't used at the moment
-			//Default:          nil,
-			// TODO exposed when other #3238 is merged
-			//Description:      "",
+			IsTypeHint:  field.ContainsDiscriminatedTypeValue,
+			JsonName:    field.JsonName,
+			Optional:    field.Optional,
+			Required:    field.Required,
+			Description: field.Description,
+			// TODO this field is still missing - where is this information taken from?
 			//ForceNew:         false,
```

Comment:
This isn't actually used today - so we should be able to remove this one.

This stems from a period where we were investigating pushing the Terraform details into the SDK models, but that design changed and this wasn't cleaned up - so this remnant can be safely removed